VSOCK-VHOST changes

arminn opened the pull request.

arminn committed:
Revert "Use default vehicle HAL implementation"

This reverts commit 31e3459a791ef8fd2250e6a39e89baf918a57a05.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>
Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>

arminn committed:
Build lisot tool

Build lisot tool to be able to test vhsot-vsock.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>
Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>

rshym reviewed (approved):
`Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>`

arminn committed:
rcar-image-adas: add support for building DomD Google services

Provide support to build DomD backend Google services as part of the
rcar-image-adas image.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>
Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>

arminn committed:
google-trout-agl-services: fix compilation

With the Yocto update, stricter compilation rules were introduced.
Suppress these rules since the target source code is external.

Signed-off-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>
Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>

rshym commented:
`Acked-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>`

arminn commented:
Ack added
